### PR TITLE
Fixing error popup "not able to load object details"

### PR DESCRIPTION
**Pull request by ravirk91**
BugFix-6444-Fixing error popup "not able to load object details"

### CODE REVIEW

File: Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs [hunk 1]
```diff
@@ -157,12 +157,16 @@ private void InitActionsGrid()
...
         private void AddToPageList()
         {
-            ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
-            if (tempPage == null) App.PageList.Add(this);
-            else
+            if (App.PageList != null)
             {
-                App.PageList.Remove(tempPage);
-                App.PageList.Add(this);
+                ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
```
Comment (MeniKadosh1) [inline Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs]: but why App.PageList comes null now while before it was not happening?

Comment (ravirk91) [reply Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs]: In currernt branch, set value statement has removed for App.PageList. 
Previously we were setting the value in App.PageList from MainWindow.xaml.cs file. now that reference has been removed. 
image

File: Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs [hunk 2]
```diff
@@ -155,16 +154,20 @@ private void InitActionsGrid()
...
+        //        ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
+        //        if (tempPage == null) App.PageList.Add(this);
+
+        //        else
+        //        {
+        //            App.PageList.Remove(tempPage);
+        //            App.PageList.Add(this);
+        //        }
+        //    }
+        //}
```
Comment (MeniKadosh1) [inline Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs]: delete the commented code or Codacy will mark it as issue

Comment (ravirk91) [reply Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs]: commented code has been deleted.

**Pull request merged**